src/main/java/org/broadinstitute/hellbender/utils/SequenceDictionaryUtils.java

```diff
@@ -50,9 +50,11 @@
     public enum SequenceDictionaryCompatibility {
         IDENTICAL,                      // the dictionaries are identical
         COMMON_SUBSET,                  // there exists a common subset of equivalent contigs
+        SUPERSETS,               // the second dict's set of contigs is a superset of the first dict's; used
+        // specifically with reference and reads
```

Comment:
Should also use the singular `SUPERSET` here for consistency with the other existing enum values (eg., `COMMON_SUBSET`)